### ISSUE TITLE
Add complementary Dangit, git! site

### ISSUE THREAD

**Issue by erictleung** [freeCodeCamp/how-to-contribute-to-open-source#306]
There is already [Oh shit, git!](http://ohshitgit.com/) in the README, but there is a exact copy of the site without the subjectively offensive word.

So add this site along site the above link http://dangitgit.com/

It can be something like this

```
- [Oh shit, git!](http://ohshitgit.com/) - how to get out of common git mistakes described in plain English; also see [Dangit, git!](http://dangitgit.com/) for the page without swears.
```

**Comment by AkankshaGupta97** [comment]
I would like to do it.

**Comment by erictleung** [comment]
Closed by  https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/307

**Comment by erictleung** [comment]
@AkankshaGupta97 thanks for your interest! Unfortunately, someone else already made a pull request for this. Feel free to take a look at the other issues so other opportunities. Happy coding!
